### PR TITLE
release 3.4.0

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by minump** [comment]
I think the security.md file and yaml files needs to be updated as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,15 +27,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix that the screen reader is not able to read the dropdown list in campus event listing page. [#959](https://github.com/rokwire/events-manager/issues/959)
 - Fix that the screen reader is not able to read options of the dropdown lists as user navigates. [#961](https://github.com/rokwire/events-manager/issues/961)
 - Fix the issue that hours and seconds will be displayed for all day events in calendar events. [#958](https://github.com/rokwire/events-manager/issues/958)
-
+- Set the empty location description if the user event is physical event and no location provided. [#981](https://github.com/rokwire/events-manager/issues/981)
 ### Changed
```
Comment (sandeep-ps) [inline CHANGELOG.md:31]: Minor thing. There is a missing space before the Changed section.

**Review by minump — APPROVED** [review]
Approving with minor changes as requested [here](https://github.com/rokwire/events-manager/pull/978#issuecomment-1293924889)

File: CHANGELOG.md (hunk 1 above)
Comment (sandeep-ps) [reply CHANGELOG.md:32]: Thanks.

File: release_base_script.sh [hunk 2]
```diff
@@ -27,5 +27,5 @@ if [[ "${GIT_BRANCH}" = "master" ]]; then
 elif [[ "${GIT_BRANCH}" = "develop" ]]; then
     VERSION="develop"
 else
-    exit 0
+    VERSION="release_3.4.0"
```
Comment (sandeep-ps) [inline release_base_script.sh:30]: This change needs to be removed.

File: release_base_script.sh [hunk 3]
```diff
@@ -26,6 +26,4 @@ if [[ "${GIT_BRANCH}" = "master" ]]; then
     VERSION=${VERSION:-"$(git describe --abbrev=0 --tags)"}
 elif [[ "${GIT_BRANCH}" = "develop" ]]; then
     VERSION="develop"
```
Comment (sandeep-ps) [inline release_base_script.sh:28]: It looks like the original code:
```
else
    exit 0
```
has been removed. If it was on purpose, it's fine. I just wanted to make sure that it was intended. Thanks.

Comment (sandeep-ps) [reply release_base_script.sh:28]: So, you seem to have deleted the else condition when you reset the changes. Please see the diff in GitHub when you get a chance. I was talking about that in my message above.

**Comment by bingzhang** [comment]
@sandeep-ps can you check the comments I resolved? then we can merge it. thanks

File: release_base_script.sh (hunk 3 above)
Comment (minump) [inline release_base_script.sh:28]:
```suggestion
    VERSION="develop"
else
    exit 0
```

**Comment by minump** [comment]
Approved for merge to develop

File: release_base_script.sh (hunk 3 above)
Comment (sandeep-ps) [reply release_base_script.sh]: The changes look good. Resolving this conversation. Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**